Tests/PrettyPrinterTests/AttributeTests.swift

```diff
@@ -0,0 +1,24 @@
+public class AttributeTests: PrettyPrintTestCase {
+  public func testAttributeParamSpacing() {
+    let input =
+      """
+      @available
+      @available(iOS 9.0, *)
+      @available(*, unavailable, renamed: "MyRenamedProtocol")
+      @available(iOS 10.0, macOS 10.12, *)
+      """
+
+    let expected =
+      """
+      @available
+      @available(iOS 9.0, *)
+      @available(*, unavailable, renamed: "MyRenamedProtocol")
+      @available(iOS 10.0, macOS 10.12, *)
+
+
```

Comment:
Why is there an extra blank line in the expected output? Is this just a weird artifact of the attributes being in isolation and not attached to a decl?

Reply:
Yes, this is an artifact of not being attached to a decl, so this isn't a *real* test per se. Its purpose is to test the spacing between the tokens in the argument lists.